src/templates/cmp.js

```diff
@@ -213,4 +223,56 @@
   } else {
     loadTcfapi(3);
   }
+
+  // send device ids
+  function sendDeviceId(consent, retriesLeft) {
+    if (retriesLeft < 0) {
+      return;
+    }
+
+    if (!__hbb_tracking_tgt || !__hbb_tracking_tgt.getDID) {
+      setTimeout(function () {
+        sendDeviceId(consent, retriesLeft - 1);
+      }, 100);
+      return;
+    }
+
+    __hbb_tracking_tgt.getDID(function (deviceId) {
+      var image = document.createElement('img');
```

Comment:
I would not consider this necessary as the created `img` tag is not being attached to the DOM. If we want to do this as a precaution anyway, I would do it in a separate PR, because we then also need to do it for the images used in `cmpapi.js`